client/my-sites/backup/rewind-flow/steps/missing-credenitals.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { backupPath, settingsPath } from 'calypso/lib/jetpack/paths';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+
+interface Props {
+	siteSlug: string | null;
+	enterCredentialsEventName: string;
+	goBackEventName: string;
+}
+
+const MissingCredentials: FunctionComponent< Props > = ( {
```

Comment:
We could start moving these flow steps into a separate components like this one.